rac_aspace/decorators.py

```diff
@@ -0,0 +1,9 @@
+
+
+def check_dictionary(func):
+    def inner(*args, **kwargs):
+        if not isinstance(args, dict):
+            print('Parameter is not a dictionary')
+            return
```

Comment:
```suggestion
            raise TypeException('{} is not a dictionary'.format(args[0])
```
Let's `raise` rather than simply printing so we can stop bad things from happening.